.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -0,0 +1,29 @@
+name: Feature Request
+description: Suggest an idea for this project
+title: "[Enhancement]: "
+labels: ["enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to request a feature!
+  - type: textarea
+    id: problem
+    attributes:
+      label: Is your feature request related to a problem?
+      description: Please describe.
+      placeholder: I'm always frustrated when...
```

Comment:
This might be silly, but should we write something more positive here? Like:
```
It would be really cool if...
```

Reply:
Absolutely - suggest it as a change here

Reply:
```suggestion
      placeholder: It would be really cool if...
```